src/test/java/core/basesyntax/strategy/PurchaseOperationStrategyTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+//import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.fruitstorge.FruitStorage;
+import core.basesyntax.fruittransaction.FruitTransaction;
+import core.basesyntax.operations.Operation;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationStrategyTest {
+    @Test
+    public void testHandleValidTransaction() {
+        // Set up initial data in FruitStorage for testing
+        OperationHandler operationHandler = new PurchaseOperationStrategy();
+        FruitStorage.fruitStorage.put("Apple", 30);
+        // Create a FruitTransaction with a valid amount
+        FruitTransaction validTransaction = new FruitTransaction(Operation.PURCHASE,"Apple", 5);
+        FruitTransaction validTransaction2 = new FruitTransaction(Operation.PURCHASE,"Apple", 10);
+        // Call the handle method
+        operationHandler.handle(validTransaction);
+        operationHandler.handle(validTransaction2);
+        // Verify that the balance is updated correctly
+        assertEquals(15, FruitStorage.fruitStorage.get("Apple"));
```

Comment:
your tests should be readable enough so that no comments are needed. At least for such simple scenario